script/cli/init.lua

```diff
@@ -4,13 +4,13 @@ if _G['VERSION'] then
 end
 
 if _G['CHECK'] then
-    require 'cli.check'
-    os.exit(0, true)
+    local checkPassed = require 'cli.check'
+    os.exit(checkPassed and 0 or 1, true)
 end
 
 if _G['CHECK_WORKER'] then
-    require 'cli.check_worker'
-    os.exit(0, true)
+    local checkPassed = require 'cli.check_worker'
```

Comment:
after some thought, this might be a bit anti-pattern 🤔 
- when successfully required a module, the lua loader will set the **return value of that module** into `package.loaded[moduleName] = result`
- if a module doesn't have return value, the default result will be `true`
i.e. `package.loaded['cli.check_worker] = true`
- and in general when there is return value, one would usually return a `table` representing the exports of that module (?)
- therefore here if we directly return a `checkPassed` result from the module, seems a bit anti-pattern

---

Maybe be should wrap the whole main logic inside `'cli.check'` / 'cli.check_worker' into a `runCLI()` function, just like how the other flags (DOC_UPDATE / DOC / VISUALIZE) are done 😕